src/app/utilities/api-clients/user.js

```diff
@@ -72,18 +78,56 @@ export default class user {
     }
 
     static logOut() {
-        const accessTokenCookieRemoved = cookies.remove("access_token");
-        if (!accessTokenCookieRemoved) {
-            console.warn(`Error trying to remove 'access_token' cookie`);
-            return;
+        function clearCookies() {
+            const accessTokenCookieRemoved = cookies.remove("access_token");
+            if (!accessTokenCookieRemoved) {
+                console.warn(`Error trying to remove 'access_token' cookie`);
+            }
+            if (cookies.get("collection")) {
+                cookies.remove("collection");
+            }
+            localStorage.removeItem("loggedInAs");
+            localStorage.removeItem("userType");
+            store.dispatch(userLoggedOut());
+            store.dispatch(reset());
         }
-        if (cookies.get("collection")) {
-            cookies.remove("collection");
+
+        const config = window.getEnv();
+        if (config.enableNewSignIn) {
+            const headers = {
+                Authorization: cookies.get("access_token")
+            };
+            user.expireSession(headers)
+                .then(response => {
+                    clearCookies();
+                })
+                .catch(error => {
+                    if (error.status === 400) {
+                        const notification = {
+                            type: "warning",
+                            message: "An error occurred during sign out 'InvalidToken', please contact a system administrator",
+                            isDismissable: true,
+                            autoDismiss: 20000
+                        };
+                        notifications.add(notification);
+                        console.error("Error occurred sending DELETE to /tokens/self - InvalidToken");
+                        log.event("error on sign out sending delete to /tokens/self failed with an invalid token", log.error(error));
```

Comment:
Not something for this PR but looking at this, it would be good to update the logging package we use in Florence to align with our logging standards with the `log.go` v2 changes. We could also then maybe introduce new functions to also append warnings/errors etc. to the console rather than having to make two separate calls like here

Reply:
Agreed, raised a ticket in the backlog for this: https://trello.com/c/kHlQt53r/2110-florence-js-logging-needs-to-be-updated-to-be-in-line-with-current-logging-standards